python/vtool/houdini_lib/core.py

```diff
@@ -7,3 +10,43 @@ def save(filepath):
 
 def load(filepath):
     hou.hipFile.load(filepath)
+
+
+def merge(filepath):
+    hou.hipFile.merge(filepath)
+
+def export_nodes(path, nodes):
+
+    util_file.refresh_dir(path, delete_directory=False)
+
+    parent = nodes[0].parent()
+    parent_path = nodes[0].path()
+    parent_path = util_file.get_dirname(parent_path)
+    if parent_path.startswith('/'):
+        parent_path = parent_path[1:]
+
+    parent_path += '/nodes.cpio'
+
+    filepath = util_file.join_path(path, parent_path)
+    folder_path = util_file.get_dirname(filepath)
+    folder_path = util_file.create_dir(folder_path)
+
+    parent.saveItemsToFile(nodes, filepath)
+
+
+def import_nodes(path, context=None):
+
+    folders = util_file.get_folders(path, True, skip_dot_prefix=True)
+    for folder in folders:
```

Comment:
The first iterator here should be broken apart from the nested second and should be its own function. You've got a function here that's sole responsibility should be getting nodes, yet it's climbing a file structure tree, performing path operations, and retrieving files.

Reply:
I'm using some folder structure here to mimic what's happening in the houdini node hierarchy.  So that when I import the .cpio file it has some context about where it should import in Houdini.   Maybe would be better accomplished with a settings file.  I will come back to this and add a TODO comment